include/analog.h

```diff
@@ -1,6 +1,13 @@
 #ifndef __ANALOG_H__
 #define __ANALOG_H__
 
+#include <Arduino.h>
+#include "adc.h"
+#include "ina281.h"
+#include "STM32TimerInterrupt_Generic.h"
+
+#define IO_UPDATE_PERIOD 100000 // us
```

Comment:
We might need a `const.h` file since this macro will probably be used by `digital.cpp` as well.